cmd/finch/nerdctl.go

```diff
@@ -170,7 +170,11 @@ func (nc *nerdctlCommand) run(cmdName string, args []string) error {
 		envVars[evar] = eval
 	}
 
-	passedEnvs := []string{"COSIGN_PASSWORD"}
+	passedEnvs := []string{
```

Comment:
Could you add some design context in the PR description? What are the user scenarios are we trying to support here? 
Is the env var the only way to pass AWS credential? Any other possible ways (like credential file)? Do we need to support them? https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html 

Once we clarify the scope, I think there can be some docs to guide users using the feature. "ECR credential helper integration" is more like the implementation details but the user facing feature should be something like "simplify ECR interfaction in FInch". I think the documentation should focus on its usage.

Reply:
Both the env variable and the credential file are supported. I will add some documentation to the README.